lib/openstax/cnx/v1/key_term.rb

```diff
@@ -0,0 +1,32 @@
+module OpenStax::Cnx::V1
+  class KeyTerm < Element
+    MATCH_KEY_TERM = "//*[contains(@class, 'os-glossary-container')]//dl"
+
+    MATCH_KEY_TERM_TEXT = ".//dt"
+    MATCH_KEY_TERM_DESCRIPTION = ".//dd"
+
+    def initialize(node:)
+      super
+    end
+
+    def description
+      node.xpath(MATCH_KEY_TERM_DESCRIPTION).text
```

Comment:
i wonder if its possible for descriptions to have non-text content? like math or whatever? maybe this is irrelevant, because i see figure captions also use the text function

Reply:
hmm, there might be math inside: https://openstax.org/books/chemistry-2e/pages/2-key-terms
I've found it with https://content-finder.herokuapp.com/?book=Chemistry&element=dd%20math (it is using very old content but was correct for this book)
I didn't found any images

Reply:
Math in key terms can be both in inline and block form :/

Reply:
![image](https://user-images.githubusercontent.com/31478212/118648515-c5906a80-b7e2-11eb-95a3-1502e1f58966.png)

```
irb(main):007:0> cbook.root_book_part.parts[2].parts[8].elements(element_classes: [OpenStax::Cnx::V1::KeyTerm])[6].description
=> "(also, unified atomic mass unit, u, or Dalton, Da) unit of mass equal to 112112 of the mass of a 12C atom"
```